cpp/vm/evmzero/interpreter_test.cc

```diff
@@ -3198,6 +3198,47 @@ TEST(InterpreterTest, BASEFEE_PreRevision) {
   });
 }
 
+///////////////////////////////////////////////////////////
+// BLOBBASEFEE
+TEST(InterpreterTest, BLOBBASEFEE) {
+  evmc_tx_context tx_context{
+      .blob_base_fee = evmc::uint256be(42),
+  };
+
+  MockHost host;
+  EXPECT_CALL(host, get_tx_context()).Times(1).WillOnce(Return(tx_context));
+
+  RunInterpreterTest({
+      .code = {op::BLOBBASEFEE},
+      .state_after = RunState::kDone,
+      .gas_before = 10,
+      .gas_after = 8,
+      .stack_after = {42},
+      .host = &host,
+      .revision = EVMC_CANCUN,
+  });
+}
+
+TEST(InterpreterTest, BLOBBASEFEE_OutOfGas) {
+  RunInterpreterTest({
+      .code = {op::BLOBBASEFEE},
+      .state_after = RunState::kErrorGas,
+      .gas_before = 1,
+      .revision = EVMC_CANCUN,
+  });
+}
+
+TEST(InterpreterTest, DISABLED_BLOBBASEFEE_StackOverflow) {}
```

Comment:
Why is this test `DISABLED_`?

Reply:
We discussed this yesterday: all StackOverflow are disabled and/or empty. #520

Reply:
yes. But we suspect this was done as "remainder" to implement them later on. I think with the helper function I introduce, and someone can take task #520  later on and implement all the missing tests.